### ISSUE TITLE
Developers Getting started guide + vgo

### ISSUE THREAD

**Issue by NDari** [gorgonia/gorgonia#221]
This is a continuation from the new defunct #116, to move toward a uniform "getting started" experience for anyone who would like to contribute to this project.

- [x] remove `Gopkg.*` from the repo
- [x] add `go.mod`
- [x] add a section in the readme on how to build/get started with this project for developers

To reiterate, this should not interfere with keeping the package `go get`able. This is only for developers to build and test locally, as well as keeping some consistency in builds between releases as we move toward 1.0

**Comment by chewxy** [comment]
Thank you for doing this!

**Comment by NDari** [comment]
Absolutely! I am not sure when would be a good time to merge this in. In 1.11 vgo with have an experimental status, which has historically meant a full release the following cycle. If so, we are about ~7 months away from an official release.
